pages/index.tsx

```diff
@@ -249,128 +252,140 @@ export default function Home(props: {
       </section>
       {/* Featuring Keynotes speakers */}
 
-      <section className="flex overflow-x-auto bg-gray-200 min-h-[24rem]">
-        <div className="flex items-center justify-center font-bold p-6 md:text-4xl text-2xl my-4">
-          Featuring Keynote Speakers
-        </div>
-        <div className="flex flex-col justify-center py-6 md:px-6">
-          {/* Row 1 */}
-          <div className="flex">
-            {speakers.map(
-              ({ name, description, fileName }, idx) =>
-                idx < speakers.length / 2 && (
-                  <KeynoteSpeaker
-                    key={idx}
-                    name={name}
-                    description={description}
-                    cardColor={colorSchemes[idx % 3]}
-                    imageLink={fileName}
-                  />
-                ),
-            )}
+      {speakers.length != 0 && (
```

Comment:
This is probably just me being nit-picky but might wanna consider using `!==` instead of `!=`.